apps/condo/index.js

```diff
@@ -65,6 +72,11 @@ const keystone = new Keystone({
     },
 })
 
+const cache = {}
```

Comment:
its better to move it inside the initCache

Reply:
Done!